account_banking_sepa_credit_transfer/README.rst

```diff
@@ -32,7 +32,9 @@ This module is part of the OCA/bank-payment suite.
 Configuration
 =============
 
-No specific configuration.
+* You need to configure a valid BIC code on the banks belonging to the 
+source and destination bank accounts, or the export will throw a 
+not-so-informative 'Compute error'.
 
```

Comment:
If there is such an error, we should fix it at the source of the problem instead of describing it in the README.
For a SEPA credit transfer, we do need BIC on source bank account but we don't need it any more on destination bank account, even for cross-borders wire transfer... and I think it works fine in Odoo.

Reply:
I think you are right. 

After some investigation I have seen that from version `9.0` on, the whole of this module and the modules it depends on, have been refactored. Probably the error lies in this function ([8.0](https://github.com/OCA/bank-payment/blob/8.0/account_banking_pain_base/models/banking_export_pain.py#L304) vs [9.0](https://github.com/OCA/bank-payment/blob/9.0/account_banking_pain_base/models/account_payment_order.py#L361)) and more specifically [here](https://github.com/OCA/bank-payment/blob/8.0/account_banking_pain_base/models/banking_export_pain.py#L277). And I think it's already fixed from 9.0 onwards.

So I think the best solution would just be to backport the whole of it to 8.0, but that is a lot of effort. Alternatively, we could do with this README update, or a small bugfix in the 8.0 module. Let me see if I can do the latter quickly.

Reply:
Well, it was not that quickly, but at least I added support for the BIC being omitted (inspired by the 9.0 code) and wrote some tests. The real effort would be to backport everything, but at least this feels better than just adding a README comment.